src/MainWindow.vala

```diff
@@ -694,7 +694,7 @@ public class Installer.MainWindow : Gtk.Dialog {
         }
 
         stack.visible_child = encrypt_view;
-        encrypt_view.reset();
+        encrypt_view.reset ();
```

Comment:
Not sure if this impacts functionality or not, but having a space here (and on line 756) looks strange and is inconsistent with other places the reset functions are called (e.g. lines 255, 331, and 535 in this file.) Does the change to this line actually do anything?

Reply:
The space before the parenthesis is actually the correct lint for vala. It is incorrect elsewhere https://github.com/elementary/appcenter/pull/1752#discussion_r766363200

Reply:
Looks like it's part of the elementary OS code style, which they'd be following for AppCenter: https://docs.elementary.io/develop/writing-apps/code-style#whitespace 

Fair enough.